staging/src/k8s.io/apiserver/pkg/endpoints/installer.go

```diff
@@ -382,7 +382,7 @@ func (a *APIInstaller) registerResourceHandlers(path string, storage rest.Storag
 		actions = appendIf(actions, action{"LIST", resourcePath, resourceParams, namer, false}, isLister)
 		actions = appendIf(actions, action{"POST", resourcePath, resourceParams, namer, false}, isCreater)
 		actions = appendIf(actions, action{"DELETECOLLECTION", resourcePath, resourceParams, namer, false}, isCollectionDeleter)
-		// DEPRECATED
```

Comment:
Haven't been deprecated long ago?  Can we remove now?

Reply:
it wasn't formally announced, so not yet.

Reply:
ic